test/customer_repository_test.rb

```diff
@@ -1,5 +1,5 @@
-require './test/test_helper'
-require './lib/customer_repository'
+require_relative '../test/test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.